### PR TITLE
Utility program to generate a binding table

### PR DESCRIPTION
**Pull request by dpino**
Can generate a binding table based on the following parameters:

* from_ipv4: starting IPv4 address.
* from_b4: starting IPv6 B4 address.
* num_ips: number of public IPv4 addresses.
* br: border router address.
* psid_len: numbers of bits for port-set length.
* shift: number of bits for port range.

Flag _--output_ prints binding-table to a file.

### CODE REVIEW

**Comment by wingo** [comment]
Nice!  LGTM.

File: src/program/lwaftr/generate_binding_table/README [hunk 1]
```diff
@@ -0,0 +1,14 @@
+Usage:
+
+generate_binding_table <ipv4> <num_ipv4s> <br_address> <b4> <psid_len> <shift>
+    --output <filename>: Write down binding-table to file.
+
```
Comment (kbara) [inline src/program/lwaftr/generate_binding_table/README]: That has 6 arguments; the examples have 5. I assume that 'shift' is the undocumented one?

**Comment by dpino** [comment]
@kbara: right, when `shift` is omitted its value is calculated as 16 - psid_len.

**Comment by kbara** [comment]
Yes, I understand that, but we shouldn't have help that can only be understood by reading the code!

**Pull request merged**